### PR TITLE
Update README.md

### LINKED ISSUE #1: TODO
- [x] simplify everything to index.html
- [ ] use official JS library to cleanup the code
- [ ] UI (added primer CSS)
- [ ] make a logo
- [ ] show video duration & author
- [ ] does it work for any playlist? if so, rename
- [ ] Move the repo to the watchlater org
- [ ] add support for [removing videos from playlist](https://developers.google.com/youtube/v3/docs/playlistItems/delete) (will require additional permissions, hold off for now)

### LINKED ISSUE #2: YouTube API breaking change
Nothing works anymore 💀 
See the announcement here: https://developers.google.com/youtube/v3/revision_history#august-11-2016

### PR DESCRIPTION
**Pull request by dalthviz**
Fixes #2

I´m sorry it didn´t work out with the changes in the Youtube API...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 Because YouTube broke the "load more" button on the "Watch Later" page.
 
 * Sept 10, 2016 update: this appears be fixed now.
+* Oct 15, 2016 update: this project is know impossible 😢.
```
Comment (shakeelmohamed) [inline README.md]: It should be "now" instead of "know"

Comment (shakeelmohamed) [reply README.md]: Let's also add a link to the announcement from #2

**Review by shakeelmohamed — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dalthviz) [reply README.md]: ok 👍, something like this?:
- Oct 15, 2016 update: [this project is now impossible 😢.](https://developers.google.com/youtube/v3/revision_history#august-11-2016)

Comment (shakeelmohamed) [reply README.md]: looks fine

**Review by shakeelmohamed — APPROVED** [review]

**Comment by shakeelmohamed** [comment]
Great, thanks for doing this @dalthviz!

**Comment by dalthviz** [comment]
You´re welcome @shakeelmohamed 😄 👍

**Pull request merged**